recipe/meta.yaml

```diff
@@ -61,3 +61,4 @@ extra:
     - xhochy
     - jschueller
     - thewchan
+    - larsoner
```

Comment:
Hope it's okay to add me so that I can keep things like this plus https://github.com/conda-forge/pyobjc-framework-fsevents-feedstock/pull/42 and https://github.com/conda-forge/pyobjc-framework-coreservices-feedstock/pull/23 moving in the future without needing to bother other folks!